### PR TITLE
make status enums

### PR DESCRIPTION
**Pull request by arnavtaneja**
### Administrative Info
Monday Board ID: 853648344
Make sure your branch name conforms to: `<feature/staging/hotfix/...>/[username]/[Monday Item ID]-[3-4 word description separated by dashes]`. Otherwise, please rename your branch and create a new PR.

### Changes
- Made file that created status enums

### Testing
How did you confirm your changes worked? 
- I did not. Should work in theory though.

### Confirmation of Change 
Upload a screenshot, if possible. Otherwise, please provide instructions on how to see the change.
- new file in the migrations folder

| [TODO Screenshot Title] |
| --- |
| [TODO Screenshot Link] |

### CODE REVIEW

**Review by eliasfang — APPROVED** [review]
LGTM

File: app/database/migrations/20201207174436_make_status_enum.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+const status = [
+    'SCHEDULED',
+    'ON_PROCESS',
+    'SUCCESS',
+    'FAIL'
+];
+
+exports.up = function(knex) {
+    return knex.schema.table('jobs', (tbl) => {
+        tbl.string('status');
```
Comment (mingkaili) [inline app/database/migrations/20201207174436_make_status_enum.js]: This is very good, but I did more research to the syntax and the functionalities of this particular method. It is better to use the native enum method from knex and actually handle it that way. 

        `tbl.enum('status', status, {useNative: true, enumName:'status_enum'}).notNullable().index();`
Also you can delete the .then portion of the code.

Sorry for the confusion

**Review by mingkaili — CHANGES_REQUESTED** [review]
Change the enum initialization type

File: app/database/migrations/20201207174436_make_status_enum.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+
+const status = [
+    'SCHEDULED',
+    'ON_PROCESS',
+    'SUCCESS',
+    'FAIL'
+];
+
+exports.up = function(knex) {
+    tbl.enum('status', status, {useNative: true, enumName:'status_enum'}).notNullable().index();
```
Comment (mingkaili) [inline app/database/migrations/20201207174436_make_status_enum.js]: return knex.schema.table('jobs', (tbl) => {

**Review by mingkaili — APPROVED** [review]

**Pull request merged**